controllers/templatesync/template_sync.go

```diff
@@ -1310,13 +1308,7 @@ func (r *PolicyReconciler) processDependencies(
 		var res dynamic.ResourceInterface
 
 		if namespaced {
-			ns := dep.Namespace
-			if ns == "" && dep.Group == policiesv1.GroupVersion.Group {
-				// ocm policies should always be in the cluster namespace
-				ns = r.ClusterNamespace
-			}
-
-			res = dClient.Resource(rsrc).Namespace(ns)
+			res = dClient.Resource(rsrc).Namespace(dep.Namespace)
```

Comment:
Should this also use the new function?
```suggestion
			res = dClient.Resource(rsrc).Namespace(getDepNamespace(r.ClusterNamespace, dep))
```

Reply:
@dhaiducek it's not needed in this case because the input to `processDependencies` already has dependencies with the correct namespace.